src/components/ItemDetaile/details.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router';
+import { Link } from 'react-router-dom';
+import { setTitle, setActiveCoin } from '../../redux/Slice/ItemSlice';
+import './details.css';
+
+const Details = () => {
+  const dispatch = useDispatch();
+  const location = useLocation();
+  // eslint-disable-next-line
```

Comment:
Good job working on the project requirements, but I noticed you disabled eslint in this file. It is against microverse policy to disable linters so it would be nice if you enabled eslint

Reply:
Okay, Well done✔